src/h3lib/lib/baseCells.c

```diff
@@ -863,6 +863,16 @@ void _baseCellToFaceIjk(int baseCell, FaceIJK* h) {
     *h = baseCellData[baseCell].homeFijk;
 }
 
+int _baseCellToCCWrot60(int baseCell, int face) {
+    if (face > NUM_ICOSA_FACES) return INVALID_ROTATIONS;
+    for (int i = 0; i < 3; i++)
+        for (int j = 0; j < 3; j++)
+            for (int k = 0; k < 3; k++)
+                if (faceIjkBaseCells[face][i][j][k].baseCell == baseCell)
+                    return faceIjkBaseCells[face][i][j][k].ccwRot60;
```

Comment:
Please add the curly braces for this block, this looks like it could get very broken if edited without the formatting enforced.

Reply:
Oh fine :).